app/ui/omni-box.js

```diff
@@ -221,6 +251,9 @@ class OmniBox extends HTMLElement {
 }
 
 function makeHttps (query) {
+  if (looksLikeLegacySSB(query)) {
```

Comment:
Is this clause still needed?

Reply:
Nope! Funny how removing earlier hacks fixes things :rofl: